src/daemon.rs

```diff
@@ -16,6 +16,8 @@ use serde_json::{from_str, from_value, Value};
 use bitcoin::consensus::encode::{deserialize, serialize_hex};
 #[cfg(feature = "liquid")]
 use elements::encode::{deserialize, serialize_hex};
+#[cfg(feature = "tracing-enabled")]
```

Comment:
the feature name should maybe be "otlp-tracing" or "tracing". "enabled" is what we do with it.

Reply:
I changed the name